lib/cocoapods-core/specification/dsl.rb

```diff
@@ -126,6 +126,25 @@ module DSL
       root_attribute :version,
                      :required => true
 
+      #------------------#
+
+      # @!method swift_version=(version)
+      #
+      #   The version of Swift that the specification supports.
```

Comment:
Could probably expand slightly more for the end user in which we document how this is used as a "best effort" value.

Reply:
Yeah, planning on following up with more info and docs

Reply:
No longer need to do this, this is just a string.